mnemosyne/runtime_diagnostics.py

```
@@ -0,0 +1,89 @@
+    except Exception:
+        add("core", "embeddings_available", "ERROR", "embeddings capability unavailable")
+
+    try:
+        from mnemosyne.core.beam import _SQLITE_VEC_AVAILABLE
+
+        vec_can_load = False
+        if _SQLITE_VEC_AVAILABLE:
+            try:
+                test_conn = sqlite3.connect(":memory:")
+                try:
+                    test_conn.enable_load_extension(True)
```

Ruff (0.15.21)

[warning] Boolean positional value in function call

(FBT003)